src/aleph/handlers/storage.py

```diff
@@ -24,24 +27,27 @@
 LOGGER = logging.getLogger("HANDLERS.STORAGE")
 
 
-async def handle_new_storage(message, content):
+async def handle_new_storage(message: Dict, content: Dict):
     if not app["config"].storage.store_files.value:
         return True  # Ignore
 
-    item_type = content.get("item_type")
+    # TODO: ideally the content should be transformed earlier, but this requires more clean up
+    #       (ex: no more in place modification of content, simplification of the flow)
+    try:
+        store_message = StoreMessage(**message, content=content)
+    except ValidationError as e:
+        print(e)
```

Comment:
Should we raise an exception here and follow up on invalid messages reported on Sentry during testing ?

Reply:
Mmh I don't think Sentry is the right tool for this, because this is not something that we can fix. IMO right approach is to log invalid messages in a dedicated table and have some monitoring tool (like the one we discussed yesterday) pick up the info from an API.